force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -101,17 +124,18 @@ public with sharing class ClassesObjectsMethods {
      * @return The number of vowels in the string.
      */
     public static Integer countVowels(String str) {
+        
         // Initialize the result Integer
-
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
-            // if it is a vowel, increment the result Integer
+        Integer vowelCount = 0;
+        List<String> letters = str.split('');
+        List<String> vowels = new List<String>{'a', 'e', 'i', 'o', 'u'};
+
+        for (String letter : letters) {
+            if (vowels.contains(letter)) {
+                vowelCount += 1;
```

Comment:
Another way to write this is `vowelCount++`

Reply:
Great point, not sure how I missed that.